client/blocks/eligibility-warnings/index.jsx

```diff
@@ -53,34 +53,40 @@ export const EligibilityWarnings = ( {
 		<div className={ classes }>
 			<QueryEligibility siteId={ siteId } />
 
-			{ 'plugins' === context && ! hasBusinessPlan && ! isJetpack &&
+			{ ! hasBusinessPlan && ! isJetpack &&
 				<Banner
-					description={ translate( 'Please upgrade to install this plugin.' ) }
-					feature={ FEATURE_UPLOAD_PLUGINS }
+					description={ translate( 'Also get unlimited themes, advanced customization, no ads, live chat support, and more.' ) }
+					feature={ 'plugins' === context
+						? FEATURE_UPLOAD_PLUGINS
```

Comment:
I think users can't get to this screen from the install plugin flow. For example, I have a Personal site and I want to install bbPress:

![selection_101](https://cloud.githubusercontent.com/assets/4988512/24012286/bff09788-0a7d-11e7-9190-7c0a2edbd64e.png)

The "Install" button is not clickable.